pkg/virtctl/credentials/BUILD.bazel

```diff
@@ -6,9 +6,9 @@ go_library(
     importpath = "kubevirt.io/kubevirt/pkg/virtctl/credentials",
     visibility = ["//visibility:public"],
     deps = [
-        "//pkg/virtctl/credentials/add-key:go_default_library",
-        "//pkg/virtctl/credentials/remove-key:go_default_library",
-        "//pkg/virtctl/credentials/set-password:go_default_library",
+        "//pkg/virtctl/credentials/addkey:go_default_library",
+        "//pkg/virtctl/credentials/password:go_default_library",
+        "//pkg/virtctl/credentials/removekey:go_default_library",
```

Comment:
Why do you need to have `addkey` and `removekey`? I'd suggest to just have `pkg/virtctl/credentials/sshkey`, wdyt?

Reply:
Each of these packages contains a single `virtctl` command. We could merge them to a single package, but I'd prefer it to be in a different PR. This PR only fixes linter issues.